### PR TITLE
ceph.in: try harder if asok path does not exist

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by trociny** [comment]
@tchaikov need to update src/test/cli/ceph-conf/help.t (see jenkins failure).

**Comment by tchaikov** [comment]
@trociny updated and repushed.

File: src/ceph.in [hunk 1]
```diff
@@ -526,6 +528,7 @@ def ceph_conf(parsed_args, field, name):
         raise RuntimeError('unable to get conf option %s for %s: %s' % (field, name, errdata))
     return outdata.rstrip()
 
+
```
Comment (lxbsz) [inline src/ceph.in:531]: extra line ?

**Review by lxbsz — APPROVED** [review]

File: src/ceph.in (hunk 1 above)
Comment (tchaikov) [reply src/ceph.in:531]: it is for PEP8 compliance.

Comment (lxbsz) [reply src/ceph.in:531]: Okay, get it.

**Review by trociny — APPROVED** [review]
LGTM. It fixes the issue with rbd-mirror functional tests [1].

[1] https://pulpito.ceph.com/trociny-2020-12-16_07:42:27-rbd-wip-mgolub-testing-distro-basic-smithi/

**Comment by tchaikov** [comment]
changlog:

- catch `OSError` which could be raised when `os.stat()` the asok file.

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2020-12-16_15:58:13-rados-wip-kefu-testing-2020-12-16-1740-distro-basic-smithi
- https://pulpito.ceph.com/kchai-2020-12-17_12:24:37-rados-wip-kefu-testing-2020-12-17-1542-distro-basic-smithi/

**Pull request merged**